hack/verify-all.sh

```diff
@@ -18,11 +18,7 @@ set -euo pipefail
 
 readonly PKG_ROOT="$(git rev-parse --show-toplevel)"
 
-${PKG_ROOT}/hack/verify-gofmt.sh
-${PKG_ROOT}/hack/verify-govet.sh
```

Comment:
is govet check covered?

Reply:
Yes, it is enabled by default

Reply:
by `static.yaml` ? I don't see govet config in that file.

Reply:
`govet` is enabled by default in golangci-linter - https://golangci-lint.run/usage/linters/
It does not have to be explicitly mentioned, unless it has to be disabled :)